section/wallet-setup.html

```diff
@@ -0,0 +1,147 @@
+<section id="wallet-setup">
+  <h2>Wallet setup</h2>
+  <p>This section specifies how the user can set their [=wallet address=] for Web Monetization in the user agent.</p>
+
+  <section>
+  <h3>Terms</h3>
+  <dl>
+    <dt><dfn>Web Monetization page</dfn></dt>
+    <dd>
+      Refers to the browser's UI for managing Web Monetization settings, accessible via a browser specific URI.
+      <span class="note">In Chromium based browser the page can be located at `chrome://webmonetization`.</span>
+    </dd>
+    <dt><dfn data-lt="setup page">Web Monetization setup page</dfn></dt>
+    <dd>
+      Refers to the browser's UI for setting up Web Monetization on the [=Web Monetization page=]. The UI lets user specify the [=wallet address=], and the budget.
+      <span class="note">An example of such a UI could be an "Add a Wallet" button that opens a dialog for entering this information, with an "Add" button that starts the wallet connection process when clicked.</span>
+    </dd>
+    <dt><dfn>Interactive grant</dfn></dt>
+    <dd>
+      Open Payments requires outgoing payment [=grant=] requests to be interactive. When a grant request is interactive, it means explicit interaction by an individual (typically the client’s end user) is a required step in the delegation process.
+    </dd>
+    <dd>
+      After the grant request is made, the client must poll for a Grant Continuation request so the [=AS=] knows to issue an access token.
+    </dd>
+    <dt><dfn>Redirect URI</dfn></dt>
+    <dd>
+      The redirect URI is an URL the user is redirected to in order to accept or decline the outgoing payment grant ([=interactive grant=]) when conneting their [=wallet=] to the browser.
+    </dd>
+  </dl>
+  </section>
+
+  <!-- Authentication keys details live under the Authentication section; this page references the keypair generation step below. -->
+
+  <section class="algorithm">
+  <h3>Connecting a wallet</h3>
+
+  <p>The user can initiate the wallet setup from the [=setup page=]. The [=setup page=] provides
+    UI that lets the user: (1) enter a [=wallet address=]; (2) specify an initial spending budget;
+    (3) optionally enable automatic monthly renewal; and (4) after authentication {{ Storage/keys }} generation,
+    view and copy the newly generated public key so it can be registered with the connected wallet
+    for client identification.</p>
+
+  <p>To <b><em>process the wallet setup</em></b>, given a string |walletAddress:DOMString| (the [=wallet address=] provided by the user),
+    a number |budget:number| (the budget that the user wants to use for Web Monetization) and
+    a boolean |renewMonthly: boolean| (an option to renew the budget every month):</p>
+  <ol>
+    <li>
+      Let |walletAddressDetails:WalletAddressDetails| be the result of the [=send a wallet address request=] algorithm on |walletAddress|.
+    </li>
+    <li>Run the [=generate an Ed25519 key pair=] algorithm.</li>
```

Comment:
If this is run at connection time, the setup page has to show user the public key at this stage. Should this be a pre-requisite to wallet connection (like given `walletAddress`, ... and keys generated)? We definitely don't want UI concerns to dictate spec flow.